### DISCUSSION TITLE
[@types/node] Error constructor missing options for node >= 16.9.0

### DISCUSSION THREAD

**Discussion by RobvH** [DefinitelyTyped/DefinitelyTyped#61786] [Issues with a @types package] [answered]
Starting with Node 16.9.0 (See chang log: https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md#error-cause) the Error constructor optionally receives an options argument; on which a lower level Error can be chained to the `cause` key.

Eg.
```js
const error1 = new Error('Error one');
const error2 = new Error('Error two', { cause: error1 });
```

Also, see the Node docs for further proof / info:
https://nodejs.org/dist/latest-v16.x/docs/api/errors.html#class-error

> If the cause option is provided, it is assigned to the error.cause property.

The `cause: Error` property should also be exposed as an optional on the Error interface.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
@RobvH please review #61790 61790

**Reply by RobvH** [reply]
Reviewed and ✅ approved. Thanks for the blazingly fast turnaround. I'll try to offer a PR with my next issue. Appreciate all you do!
